kokoro/windows/gapid.wxs

```diff
@@ -49,15 +49,39 @@
         <Shortcut Id="ApplicationStartMenuShortcut" Name="GAPID" Description="Graphics API Debugger"
             Target="[!gapid.exe]" WorkingDirectory="APPLICATIONROOTDIRECTORY" Icon="gapid.ico" />
         <RemoveFolder Id="CleanUpShortCut" Directory="ApplicationProgramsFolder" On="uninstall" />
-        <RegistryValue Root="HKCU" Key="Software\Google\GAPID" Name="installed" Type="integer" Value="1" KeyPath="yes" />
+        <RegistryValue Root="HKCU" Key="SOFTWARE\Google\GAPID" Name="installed" Type="integer" Value="1" KeyPath="yes" />

```

Comment:
This is an odd change. Completely from memory, isn't "Software" more common? Again, IIRC, the registry is case insensitive, so may not matter.

Reply:
OK, so I changed this to match what I had below and I had it all uppercase below, because the key is actually all upper case in my registry...

However, it appears you are right, convention now is to use "Software", so I've updated it. Yes, the registry keys are case insensitive.